packages/frontend/core/src/blocksuite/presets/ai/chat-panel/chat-panel-messages.ts

```diff
@@ -182,6 +183,51 @@ export class ChatPanelMessages extends WithDisposable(ShadowlessElement) {
     }
   }
 
+  private _renderAIOnboarding() {
+    return this.isLoading ||
+      !this.host.doc.awarenessStore.getFlag('enable_ai_onboarding')
+      ? nothing
+      : html`<div
+          style=${styleMap({
+            display: 'flex',
+            gap: '8px',
+            flexDirection: 'column',
+            alignItems: 'center',
+            marginTop: '16px',
+            width: '100%',
+          })}
+        >
+          ${repeat(
+            AIPreloadConfig,
+            config => config.text,
+            config => {
+              return html`<div
+                @click=${() => config.handler()}
+                style=${styleMap({
+                  display: 'flex',
+                  height: '28px',
+                  gap: '8px',
+                  width: '85%',
+                  alignItems: 'center',
+                  justifyContent: 'start',
+                })}
```

Comment:
it may be need `cursor: pointer`

Reply:
The handler is still not implemented yet. when it's ready I will add it.